package.json

```diff
@@ -19,7 +22,8 @@
     "test": "jest",
     "test:ci": "jest --ci",
     "test:coverage": "jest --runInBand --coverage",
-    "type-check": "turbo run tsc"
+    "type-check": "turbo run tsc",
+    "chromatic": "npx chromatic --project-token=90f3c5364d09"
```

Comment:
We don't need to install Chromatic in the repo. We can just rely on the GH action to do handle running Chromatic.